bambou/session.go

```diff
@@ -307,24 +341,25 @@ func (s *Session) SaveEntity(object Identifiable) *Error {
 
 	buffer := &bytes.Buffer{}
 	if err := json.NewEncoder(buffer).Encode(object); err != nil {
-		return NewError(ErrorCodeJSONCannotEncode, err.Error())
+		return NewBambouError("", err.Error())
 	}
 
 	request, err := http.NewRequest("PUT", url, buffer)
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles